integration/localnet/Makefile

```
@@ -75,3 +75,14 @@ stop:
 .PHONY: load
 load:
 	go run --tags relic ../loader -log-level info -tps 1,0,10,0,100 -tps-durations 30s,10s,30s,10s
+
+.PHONY: tps-test
+tpsTest:
+	go run --tags relic ../benchmark -log-level info -tps 25 -tps-durations 10m
+
+.PHONY: tps-test-ci
+end2endTest:
+	make init
+	make start
+	go run --tags relic ../benchmark -ci-run -gitSha $(COMMIT_SHA)
```

We already have it defined in this makefile.
```suggestion
	go run --tags relic ../benchmark -ci-run -gitSha ${COMMIT}
```

---

fixed